switch_model/generators/core/dispatch.py

```diff
@@ -495,83 +496,86 @@ def c(func):
     )
 
 
-def graph(tools):
-    # Dispatch plots take a long time to make, we skip them if the skip_long flag is True
-    if not tools.skip_long:
-        graph_curtailment_per_tech(tools)
-        graph_total_dispatch(tools)
-        graph_hourly_dispatch(tools)
-        graph_hourly_curtailment(tools)
-
-def compare(tools):
-    if not tools.skip_long:
-        if tools.num_scenarios > 1:
-            compare_hourly_dispatch(tools)
-            compare_hourly_curtailment(tools)
-
+@graph(
+    "dispatch",
+    title="Average daily dispatch"
+)
 def graph_hourly_dispatch(tools):
     """
     Generates a matrix of hourly dispatch plots for each time region
     """
     # Read dispatch.csv
     df = tools.get_dataframe('dispatch.csv')
     # Convert to GW
-    df["DispatchGen_GW"] = df["DispatchGen_MW"] / 1000
+    df["DispatchGen_MW"] /= 1000
```

Comment:
Let's be consistent with what you have below. Instead of 1000 lets just put 1e3

Reply:
Fixed